.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,38 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/javascript-node
+{
+	"name": "Node.js",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/javascript-node:1-20-bullseye",
+	"features": {
+		"ghcr.io/devcontainers-contrib/features/npm-package:1": {}
+	},
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"jasonnutter.search-node-modules",
```

Comment:
For the extensions - is this the usual list of extensions from JS Advocacy? If there isn't a standard/base list, maybe you want to propose it to your team?

Reply:
These extensions that I iThese extensions are in accordance with the proposed project. It varies from project to project. So there is no standard. It can vary according to the needs of each project.ncluded because we are usi